### PR TITLE
Add broadcast and aux_communication channel

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
### Description

This PR adds broadcast_request and includes aux_communication channel to Cell.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/fuel/f3/cellnet/cell.py [hunk 1]
```diff
@@ -206,7 +285,7 @@ def _process_reply(self, future: StreamFuture):
         waiter.receiving_future = future
         waiter.in_receiving.set()
 
-    def register_request_cb(self, channel: str, topic: str, cb, *args, **kwargs):
+    def _register_request_cb(self, channel: str, topic: str, cb, *args, **kwargs):
```
Comment (yhwen) [inline nvflare/fuel/f3/cellnet/cell.py:288]: The register_request_cb() are being called several places. This renamed to "_register_request_cb()" will break them?

File: nvflare/fuel/f3/cellnet/cell.py [hunk 2]
```diff
@@ -140,11 +211,11 @@ def _future_wait(self, future, timeout):
...
         return True
 
-    def send_request(self, channel, target, topic, request, timeout=10.0, secure=False, optional=False):
-
-        self.logger.debug(f"send_request: {channel=}, {topic=}, {target=}, {timeout=}")
+    def _send_request(self, channel, target, topic, request, timeout=10.0, secure=False, optional=False):
 
-        if channel != CellChannel.SERVER_COMMAND:
+        self.logger.info(f"send_request: {channel=}, {topic=}, {target=}, {timeout=}")
+        if channel not in CHANNELS_TO_HANDLE:
```
Comment (yanchengnv) [inline nvflare/fuel/f3/cellnet/cell.py:217]: Do you still need this check here? I thought it's already checked by the __getattr__ method.

File: nvflare/fuel/f3/cellnet/cell.py [hunk 3]
```diff
@@ -155,31 +226,36 @@ def send_request(self, channel, target, topic, request, timeout=10.0, secure=Fal
...
+        self.logger.debug(f"{req_id=}: Waiting starts")
 
         # Three stages, sending, waiting for receiving first byte, receiving
 
         # sending with progress timeout
         self.logger.debug(f"{req_id=}: entering sending wait {timeout=}")
         sending_complete = self._future_wait(future, timeout)
         if not sending_complete:
-            self.logger.debug(f"{req_id=}: sending timeout")
+            self.logger.info(f"{req_id=}: sending timeout")
```
Comment (yanchengnv) [inline nvflare/fuel/f3/cellnet/cell.py:251]: Suggest to include the value of the "timeout" in log message.

File: nvflare/fuel/f3/cellnet/cell.py [hunk 4]
```diff
@@ -155,31 +226,36 @@ def send_request(self, channel, target, topic, request, timeout=10.0, secure=Fal
...
             return self._get_result(req_id)
         self.logger.debug(f"{req_id=}: sending complete")
 
         # waiting for receiving first byte
         self.logger.debug(f"{req_id=}: entering remote process wait {timeout=}")
         if not waiter.in_receiving.wait(timeout):
-            self.logger.debug(f"{req_id=}: remote processing timeout")
+            self.logger.info(f"{req_id=}: remote processing timeout")
```
Comment (yanchengnv) [inline nvflare/fuel/f3/cellnet/cell.py:258]: Include value of "timeout" in the log msg.

File: nvflare/fuel/f3/cellnet/cell.py [hunk 5]
```diff
@@ -188,16 +264,19 @@ def send_request(self, channel, target, topic, request, timeout=10.0, secure=Fal
         self.logger.debug(f"{req_id=}: entering receiving wait {timeout=}")
         receiving_complete = self._future_wait(r_future, timeout)
         if not receiving_complete:
-            self.logger.debug(f"{req_id=}: receiving timeout")
+            self.logger.info(f"{req_id=}: receiving timeout")
```
Comment (yanchengnv) [inline nvflare/fuel/f3/cellnet/cell.py:267]: Include value of "timeout" in log msg.

**Review by yanchengnv — CHANGES_REQUESTED** [review]
Looks good in general. Added a few comments/suggestions.

File: nvflare/fuel/f3/cellnet/cell.py (hunk 3 above)
Comment (IsaacYangSLA) [reply nvflare/fuel/f3/cellnet/cell.py]: Added in the new commit.

File: nvflare/fuel/f3/cellnet/cell.py (hunk 5 above)
Comment (IsaacYangSLA) [reply nvflare/fuel/f3/cellnet/cell.py]: Added in the new commit.

File: nvflare/fuel/f3/cellnet/cell.py (hunk 4 above)
Comment (IsaacYangSLA) [reply nvflare/fuel/f3/cellnet/cell.py]: Added in the new commit

File: nvflare/fuel/f3/cellnet/cell.py (hunk 2 above)
Comment (IsaacYangSLA) [reply nvflare/fuel/f3/cellnet/cell.py]: Thanks.  Forgot to remove it.  Fixed in the new commit.

**Review by yanchengnv — APPROVED** [review]
LGTM

**Review by yhwen — APPROVED** [review]
LGTM

**Review by nvidianz — APPROVED** [review]

**Pull request merged**